hack/update-codegen.sh

```diff
@@ -28,7 +28,8 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${CODEGEN_PKG}/generate-groups.sh "defaulter,deepcopy,client,informer,lister" \
+# Notice: The code in code-generator does not generate defaulter by default.
```

Comment:
Maybe add the defaulter to update-codegen.sh? 
https://github.com/kubeflow/pytorch-operator/blob/master/hack/update-codegen.sh#L36

Reply:
Thanks for your advice I have asked the upstream how to deal with the defaulter-gen and if they do not reply I will add it to update-codegen